### PR TITLE
680 - Fix Small User Manual Issues

### PR DESCRIPTION
**Pull request by settermjd**
In response to #680, this PR makes a number of small changes to fix small mistakes in the user manual.

### CODE REVIEW

File: modules/user_manual/pages/files/access_webdav.adoc [hunk 1]
```diff
@@ -100,10 +100,10 @@ The resulting dialog should appear with WebDAV already selected.
 . If WebDAV is not selected, select it.
 . Click btn:[Next].
 . Enter the following settings:
-* Name: The name you want to see in the *Places* bookmark, for example ownCloud.
-* User: The ownCloud username you used to log in, for example admin.
-* Server: The ownCloud domain name, for example *example.com* (without **http://** before or directories afterwards).
-* Folder: Enter the path `owncloud/remote.php/dav/files/USERNAME/`.
+* *Name:* The name you want to see in the *Places* bookmark, for example ownCloud.
+* *User:* The ownCloud username you used to log in, for example admin.
+* *Server:* The ownCloud domain name, for example *example.com* (without **http://** before or directories afterwards).
```
Comment (mmattel) [inline modules/user_manual/pages/files/access_webdav.adoc]: Q: `without **http://**` is this limited to http only or could it be `without **http(s)://**`

File: modules/user_manual/pages/files/encrypting_files.adoc [hunk 2]
```diff
@@ -56,13 +56,14 @@ may enable; see the Recovery Key Password section (below) to learn about this.
...
-download the files. They will see a yellow warning banner that says
-`Encryption App is enabled but your keys are not initialized, please
-log-out and log-in again.`
+Only users who have private encryption keys have access to shared encrypted files and folders. 
+Users who have not yet created their private encryption keys will not have access to encrypted shared files;
+they will see folders and filenames, but will not be able to open or download the files. 
+They will see a yellow warning banner that says:
+
+""
+Encryption App is enabled but your keys are not initialized, please log-out and log-in again.`
```
Comment (mmattel) [inline modules/user_manual/pages/files/encrypting_files.adoc]: `have not yet created their private encryption keys` is not congruent with `keys are not initialized`
(created vs. initialized)

File: modules/user_manual/pages/files/access_webdav.adoc [hunk 3]
```diff
@@ -100,10 +100,10 @@ The resulting dialog should appear with WebDAV already selected.
...
+* *Folder:* Enter the path `owncloud/remote.php/dav/files/USERNAME/`.
```
Comment (mmattel) [inline modules/user_manual/pages/files/access_webdav.adoc]: regarding the url, pls check https://github.com/owncloud/docs/pull/1822

Comment (settermjd) [reply modules/user_manual/pages/files/access_webdav.adoc]: Thanks for spotting that.

File: modules/user_manual/pages/files/access_webdav.adoc (hunk 1 above)
Comment (settermjd) [reply modules/user_manual/pages/files/access_webdav.adoc]: I just checked this. It's correct.

**Comment by settermjd** [comment]
@mmattel, just made a stack of small changes to bring this PR up to speed.

**Comment by mmattel** [comment]
Is backporting required?

**Pull request merged**